src/providers/native/index.ts

```diff
@@ -58,7 +59,10 @@ export default class NativeProvider implements Provider {
 	async createProcess(): Promise<ChildProcess> {
 		const { runtime, params, region, version, extractedDir } = this.lambda;
 		const binDir = join(runtime.cacheDir, 'bin');
-		const bootstrap = join(runtime.cacheDir, 'bootstrap');
+		const bootstrap = join(
+			runtime.cacheDir,
+			isWin ? 'bootstrap.js' : 'bootstrap'
```

Comment:
Why not always use `bootstrap.js`? Why is Windows different here?

Reply:
This is so bizarre that it would definitely warrant a comment

Reply:
Basically there's two implementations of these "bootstrap" logic. Ideally just temporarily, and we use the .js ones for all platforms, but for now running the .js ones on unix was causing tests to fail. So consider this just an in-progress transition to that while we get Windows initially/incrementally working.